playbooks/02 - Use Case - CICD/Pull Latest Content from GitHub.json

```diff
@@ -802,6 +802,15 @@
         }
     ],
     "routes": [
+        {
+            "@type": "WorkflowRoute",
+            "name": "Start -> Check Status",
+            "targetStep": "\/api\/3\/workflow_steps\/40f2c261-ea90-43e7-b7be-9e9a00685c7d",
+            "sourceStep": "\/api\/3\/workflow_steps\/9c013279-76d9-41bf-a80d-be525c0d9a43",
+            "label": null,
+            "isExecuted": false,
+            "uuid": "2e71c1db-0ac8-4ce0-a677-aca8bb3ef815"
+        },
         {
             "@type": "WorkflowRoute",
             "name": "Copy of Get Import Job -> Get Import Job",
```

Comment:
Copy of Get Import Job -> Get Import Job --> you might want to change the step name in json.

Reply:
Step names are correct in playbook, but i have created the one step by copying the already exist thats why the route have that name even though the step name is changed

Reply:
As discussed, please file a mantis for this.